terraform/variables.tf

```diff
@@ -59,3 +59,10 @@ variable "endpoint_bindings" {
   type        = map(string)
   default     = {}
 }
+
```

Comment:
QQ - why have a TLS variable and the tls integration? Isn't that counterintuitive?

Reply:
The TLS resource + integration are optional and conditioned to the `tls` variable to be `true`, meaning only if the tls flag is set to true would the self-signed-certs app be deployed and integrated to the dashboards
e.g:
```
resource "juju_application" "self-signed-certificates" {
  for_each = var.tls ? { "deployed" = true } : {}

resource "juju_integration" "tls-opensearch_dashboards_integration" {
  for_each = var.tls ? { "deployed" = true } : {}
```